packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -70,8 +81,8 @@ import 'package:flutter/src/widgets/widget_inspector.dart';
 // WidgetInspectorService show up as in scope as far as the analysis server is
 // concerned.
 extension WidgetInspectorServicePrivateMethods on WidgetInspectorService {
-  Map<String, Object> _nodeToJson(
-    DiagnosticsNode node,
+  Map<String, Object /*?*/ > /*?*/ _nodeToJson(
```

Comment:
This and below would it be easier to find the null safety code by using a TODO or NNBD type of line comment e.g.,
 // NNBD (jacob):
// Map<String, Object?>?_nodeToJson(
// DiagnosticsNode? node,
// Pre-NNBD(jacob):
Map<String, Object> _nodeToJson(
DiagnosticsNode node,

Reply:
I agree that is a bit cleaner. I'll add that refactor in the morning.